MediaWikiKit/MediaWikiKit/MWKSavedPageEntry.m

```diff
@@ -82,15 +82,17 @@ - (BOOL)isEqual:(id)object {
 }
 
 - (BOOL)isEqualToEntry:(MWKSavedPageEntry*)rhs {
-    return WMF_RHS_PROP_EQUAL(title, isEqualToTitle:);
+    return WMF_RHS_PROP_EQUAL(title, isEqualToTitle:)
+           && self.didMigrateImageData == rhs.didMigrateImageData;
```

Comment:
We covered this during review the other week… do we really want equality to fail for any other reason than the title being different? I thought we resolved this and agreed that we did not. 

Since you are changing it back, I am assuming you feel differently now? This feels a lot like we are repeating ourselves again during code review. 

This also appears to have little to do with the bug being addressed.